tests/test_plugin_loader.py

```
@@ -4463,5 +4748,171 @@ def test_update_existing_repo_to_branch_or_tag_fetch_failure(
+                caught_exceptions.append(e)
+            return mod_name
+
+        # Clean up any leftover entries from previous runs
+        for name in module_names:
+            sys.modules.pop(name, None)
+
+        try:
+            with concurrent.futures.ThreadPoolExecutor(max_workers=4) as pool:
+                futures = [pool.submit(_load_module, name) for name in module_names]
+                _done, _not_done = concurrent.futures.wait(futures, timeout=5.0)
+                self.assertFalse(
```

Ruff (0.15.7)

[warning] Use a regular `assert` instead of unittest-style `assertFalse`

Replace `assertFalse(...)` with `assert ...`

(PT009)